File: day 06/Selwin - Java/advent-of-code/src/main/java/com/selwin/Day6.java
package com.selwin;

import java.io.IOException;
import java.net.URISyntaxException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.util.List;
import java.util.stream.Collectors;

public final class Day6 {
    public static void main(String[] args) throws URISyntaxException, IOException {
        Path path = Path.of(Day6.class.getClassLoader().getResource("input.txt").toURI());
        List<String> input = Files.lines(path).collect(Collectors.toList());System.out.println(BrokenCommSystemAddon.parseElvenGibberishForStartPoint(input.get(0),4));
        System.out.println(BrokenCommSystemAddon.parseElvenGibberishForStartPoint(input.get(0),14));
    }
}

File: day 06/Toon D - Python/day6.py
line = open('input.txt').readlines()[0]

i = 0
while True:
    if len(set(line[i:i+4])) == 4:
        print(i+4)
        break
    i+=1

i = 0
while True:
    if len(set(line[i:i+14])) == 14:
        print(i+14)
        break
    i += 1
